### ISSUE TITLE
[Enhancement] Allow auto-disperse after sync when auto-reschedule is disabled

### ISSUE THREAD

**Issue by user1823** [open-spaced-repetition/fsrs4anki-helper#271]
An alpha release of AnkiDroid is soon (likely within 2-3 days) going to support native FSRS.

After that release, people won't be needing auto-reschedule after sync _that_ much. But auto-disperse would still be required.

So, I think that it is the time to modify auto-disperse such that it works after sync even if auto-reschedule is disabled.

**Comment by L-M-Sherlock** [comment]
It's pretty easy. Just rewrite this code:

https://github.com/open-spaced-repetition/fsrs4anki-helper/blob/565701a91c5278f59e2a1e11afd8dbe666ea83d4/sync_hook.py#L11-L50

I will update the add-on after the alpha release.

**Comment by user1823** [comment]
The alpha version has been released: https://github.com/ankidroid/Anki-Android/releases/tag/v2.17alpha2

**Comment by L-M-Sherlock** [comment]
I plan to separate the auto reschedule after sync and auto disperse after sync. What do you think about that?

**Comment by user1823** [comment]
That's what I suggested.

A small suggestion: If both are enabled, finding the cards with reviews should happen only once (just like it currently happens).

**Comment by L-M-Sherlock** [comment]
Patch: [fsrs4anki-helper.zip](https://github.com/open-spaced-repetition/fsrs4anki-helper/files/13197971/fsrs4anki-helper.zip)

**Comment by user1823** [comment]
Something is wrong. 

In my config, auto-reschedule is disabled.
<img width="221" alt="image" src="https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/92206575/019e61f7-3ff2-4d1a-a771-7fe1f88d0900">

But, the add-on menu shows that auto-reschedule is enabled. Also, the tooltip after sync says "x cards rescheduled …"
<img width="266" alt="image" src="https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/92206575/cc33e12b-25a2-4868-abc7-292be9da6ca8">

I have restarted Anki 2 times but that also didn't help.

**Comment by L-M-Sherlock** [comment]
I can't reproduce this bug. Could you reinstall the add-on?

**Comment by user1823** [comment]
> I can't reproduce this bug. Could you reinstall the add-on?

I was probably because of the meta.json file which was not present in the above zip and, thus, was retained in my add-on folder.

I have tested the patch and it seems to work well.

However, the tooltip is not reliably displayed. That is, after sync, the tooltip is sometimes not displayed. This issue is not specific to this patch, and I have been experiencing it since many days (probably started after updating to 23.10).

**Comment by L-M-Sherlock** [comment]
> However, the tooltip is not reliably displayed. That is, after sync, the tooltip is sometimes not displayed. This issue is not specific to this patch, and I have been experiencing it since many days (probably started after updating to 23.10).

OK. I will merge at first, and open a new issue to track it.
